lib/float.fz

```diff
@@ -55,6 +55,21 @@ float : numeric is
   redef infix ** (other float.this.type) float.this.type is abstract
 
 
+  # comparision
+  #
+  # This provides comparision operators using IEEE semantics.
+  #
```

Comment:
maybe would be good to document that `NaN` compared to anything yields false. 

I have just realized that this implementation of `>` and `<` is wrong since, e.g., `NaN > 0` and `3.14 < NaN` would both result in `true`.